tests/vmi_multus_test.go

```diff
@@ -580,6 +580,37 @@ var _ = Describe("SRIOV", func() {
 
 	tests.BeforeAll(func() {
 		tests.BeforeTestCleanup()
+		// Check if the hardware supports SRIOV
+		nodes, err := virtClient.CoreV1().Nodes().List(metav1.ListOptions{})
```

Comment:
I would rather use nodes := tests.GetAllSchedulableNodes(virtClient)  (https://github.com/kubevirt/kubevirt/blob/master/tests/vmi_lifecycle_test.go#L865)

Reply:
+1

This is a very important distinction. If a node is not schedulable, it should not be considered a candidate for running tests on. It sounds somewhat unlikely in the case of SRIOV, but this pattern has come up before. e.g. CPU count.

Reply:
Made the change as suggested